ocaml/tapctl/tapctl.mli

```diff
@@ -91,3 +91,5 @@ exception Not_a_device
 
 val of_device : context -> string -> t
 (** Given a path to a device, return the corresponding tap information *)
+
+val find_by_pid_and_minor : context -> int -> int -> t
```

Comment:
I would suggest to use labeled arguments to avoid confusion around arguments of the same type.

Reply:
Should result in more succint code as well:
```suggestion
val find : context -> pid:int -> minor:int -> t
```